payments-ui-core/src/main/java/com/stripe/android/ui/core/BillingDetailsCollectionConfiguration.kt

```diff
@@ -1,14 +1,12 @@
 package com.stripe.android.ui.core
 
 import android.os.Parcelable
-import androidx.annotation.RestrictTo
 import kotlinx.parcelize.Parcelize
 
 /**
  * Configuration for how billing details are collected during checkout.
  */
 @Parcelize
-@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 data class BillingDetailsCollectionConfiguration(
```

Comment:
Can we make this a non data class?

Reply:
What's the problem with it being a data class? `PaymentSheet.Configuration` is already a data class. All I know about this topic is that it makes java interop harder, but if the main Configuration is already data I don't know if this would be an improvement.

Reply:
We prefer new things to _not_ be a data class to help with forward binary compatibility. https://paper.dropbox.com/doc/Elements-Android-Decision-Log--B2JGBMbFJDe7wGBw54Q2xRxNAg-lIW9j6bAEBWFL0YriPLLt#:uid=732679468851952864428766&h2=Use-data-classes-for-internal-

We can't get rid of all of our existing data classes, since it would break binary compatibility.

Reply:
spoke offline, leaving as data class